Categories/Category/Monoidal/Braided.agda

```diff
@@ -37,9 +34,6 @@ record Braided : Set (levelOfTerm M) where
     B : ∀ {X Y} → X ⊗₀ Y ⇒ Y ⊗₀ X
     B {X} {Y} = braiding.⇒.η (X , Y)
 
-    B-commute : ∀ {X Y Z W} → {f : X ⇒ Y} → {g : Z ⇒ W} → CommutativeSquare (f ⊗₁ g) B B (g ⊗₁ f)
-    B-commute {_} {_} {_} {_} {f} {g} = braiding.⇒.commute (f , g)
-
```

Comment:
I would argue that B-commute could stay in here as it's quite a useful little thing to have. It's shorter than braiding.⇒.commute and doesn't require f and g to be given explicitly (which we would rarely do anyway).
I'd say it's also on par with B.

Reply:
That's fine with me, but at the moment, it's a private definition, and it's unused in this module. To be useful, it should at least be made public.